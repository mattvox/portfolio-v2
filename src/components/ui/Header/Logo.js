import React from 'react';
import styled from 'styled-components';

const LogoWrapper = styled.div`
  width: 90px;
  height: 90px;
  margin: 2rem auto 0;
  display: flex;
  align-items: center;
  position: relative;
  background: lightblue;
  color: white;
  text-decoration: none;
  cursor: pointer;
`;

const LogoText = styled.h1`
  margin: 0 auto;
`;

const Logo = () => (
  <LogoWrapper>
    <LogoText>M</LogoText>
  </LogoWrapper>
);

export default Logo;
