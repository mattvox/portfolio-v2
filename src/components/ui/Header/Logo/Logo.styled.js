import styled from 'styled-components';

export const LogoWrapper = styled.div`
  width: 90px;
  height: 90px;
  margin: 2rem auto 0;
  display: flex;
  align-items: center;
  position: relative;
  background: lightblue;
  color: white;
  text-decoration: none;
  cursor: pointer;
`;

export const LogoText = styled.h1`
  margin: 0 auto;
`;
