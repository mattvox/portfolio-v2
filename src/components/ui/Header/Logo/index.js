import React from 'react';

import { LogoWrapper, LogoText } from './Logo.styled';

const Logo = () => (
  <LogoWrapper>
    <LogoText>M</LogoText>
  </LogoWrapper>
);

export default Logo;
