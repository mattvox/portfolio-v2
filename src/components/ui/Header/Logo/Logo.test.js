import React from 'react';

import Logo from '../Logo';

describe('Logo', () => {
  it('should render correctly', () => {
    const output = shallow(<Logo />);

    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
