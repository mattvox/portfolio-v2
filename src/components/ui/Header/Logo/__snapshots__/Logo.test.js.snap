// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render correctly 1`] = `
<styled.div>
  <styled.h1>
    M
  </styled.h1>
</styled.div>
`;
